indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -49,6 +50,9 @@
       :else
       (errors/internal-error! (str "Unknown spatial representation [" coord-sys "]")))))
 
+(defn- update-values [m f & args]
+  (reduce (fn [r [k v]] (assoc r k (apply f v args))) {} m))
```

Comment:
Nit: using `reduce-kv` instead of `reduce` here will allow you to avoid destructuring the key/value pair in the anonymous function.